app/platform/forms.py

```diff
@@ -23,3 +23,25 @@ class CollectionForm(FlaskForm):
     )
     grant_id = HiddenField()
     submit = SubmitField(widget=GovSubmitInput())
+
+
+class SectionForm(FlaskForm):
+    title = StringField(
+        "Section title",
+        validators=[DataRequired("Enter a section title")],
+        filters=[lambda x: x.strip() if x else x],
+        widget=GovTextInput(),
+    )
+    collection_id = HiddenField()
+    submit = SubmitField(widget=GovSubmitInput())
+
+
+class FormForm(FlaskForm):
+    title = StringField(
+        "Form title",
+        validators=[DataRequired("Enter a form title")],
+        filters=[lambda x: x.strip() if x else x],
```

Comment:
Probably about time for us to just make this a function rather than a lambda we copy and paste everywhere